modules/packages/LinearAlgebra.chpl

```diff
@@ -221,6 +222,267 @@ class LinearAlgebraError : Error {
     }
 }
 
+/*
+This class promotes lazy computation of required
+Helper matrices to evaluate matrix exponential.
+
+High precision of expm requires smaller computations,
+hence we do a lazy computation depending on the norm value
+of the matrix.
+
+*/
```

Comment:
This seems like an implementation detail and thus maybe something that should be marked with `pragma "no doc"`?